docs/install_guides/installing-openvino-brew-linux.md

```diff
@@ -0,0 +1,114 @@
+# Install OpenVINO™ Runtime on Linux via HomeBrew {#openvino_docs_install_guides_installing_openvino_brew_linux}
+
+With the OpenVINO™ 2022.3 release, you can install OpenVINO Runtime on Linux via [Homebrew](https://brew.sh/).
+
+> **NOTE**: Only CPU is supported for inference with OpenVINO that is installed via HomeBrew.
+
+Installing OpenVINO Runtime from HomeBrew is recommended for C++ developers. If you are working with Python, the PyPI package has everything needed for Python development and deployment on CPU and GPUs. See the [Install OpenVINO from PyPI](installing-openvino-pip.md) page for instructions on how to install OpenVINO Runtime for Python using PyPI.
+
+> **NOTE**: Since the OpenVINO™ 2022.1 release, the following development tools: Model Optimizer, Post-Training Optimization Tool, Model Downloader and other Open Model Zoo tools, Accuracy Checker, and Annotation Converter can be installed via [pypi.org](https://pypi.org/project/openvino-dev/) only.
+
+See the [Release Notes](https://www.intel.com/content/www/us/en/developer/articles/release-notes/openvino-2022-3-lts-relnotes.html) for more information on updates in the latest release.
+
+
+@sphinxdirective
+.. tab:: System Requirements
+
+   | Full requirement listing is available in:
+   | `System Requirements Page <https://www.intel.com/content/www/us/en/developer/tools/openvino-toolkit/system-requirements.html>`_
+
+.. tab:: Processor Notes
+
+  Processor graphics are not included in all processors. 
+  See `Product Specifications`_ for information about your processor.
+  
+  .. _Product Specifications: https://ark.intel.com/
+
+.. tab:: Software
+
+  * [Homebrew](https://brew.sh/)
+  * `CMake 3.13 or higher, 64-bit <https://cmake.org/download/>`_
+  * GCC 7.5.0 (for Ubuntu 18.04) or GCC 9.3.0 (for Ubuntu 20.04)
+  * `Python 3.7 - 3.10, 64-bit <https://www.python.org/downloads/>`_
+
+@endsphinxdirective
+
+## Installing OpenVINO Runtime
+
+### <a name="install-openvino"></a>Step 1: Download and Install the OpenVINO Core Components
+
+@sphinxdirective
+
+1. Make sure that you have installed HomeBrew on your system. If not, you can run the following command in a prompt terminal window to install it:
+
+   .. code-block:: sh
+
+      /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
+
+2. Run the following command to install OpenVINO Runtime:
+
+   .. code-block:: sh
+
+      brew install openvino
+
+   This will install OpenVINO to this directory: `<placehoder>`. 
```

Comment:
placeholder and steps 3, 4 are not needed. Once we executed `brew install openvino` - no extra steps are required. 

brew installs packages to its dedicated folder